File: c/q3.c
#include<stdio.h>
int main()
{
	int r,c;
	printf("Enter rows and column");
	scanf("%d %d",&c, &r);
	int a[r][c];
	printf("Enter elements of array");
	printf("\n");
	for(int i=0;i<r;i++)
	{
		for(int j=0;j<c;j++)
		{
			scanf("%d",&a[i][j]);
		}
	}
  printf("original matrix");
	for(int i=0;i<r;i++)
	{
		for(int j=0;j<c;j++)
		{
			printf("%d ",a[i][j]);
		}
		printf("\n");
	}
  int ar[r][c];
	for(int i=0;i<r;i++)
	{
		for(int j=0;j<c;j++)
		{
			ar[i][j]=ar[(r-1)-j][i];
		}
	}
	printf("matrix after rotation\n");
	for(int i=0;i<r;i++)
	{
		for(int j=0;j<c;j++)
		{
			printf("%d ",ar[i][j]);
		}
		printf("\n");
	}
	
}

File: python/q3.py
n=int(input("Enter the limit:"))
a=[]
ar=[]
for i in range(0,n):
    l=[]
    for j in range(0,n):
        num=int(input("Enter the elements:"))
        l.append(num)
    arr.append(l)
z=0
for i in range(0,n):
    for j in range(0,n):
        print(a[i][j]," ",end="")
    print("\n")
for i in range(0,n):
    l=[]
    for j in range(0,n):
        l.append(a[(n-1)-j][i])
    ar.append(l)
print("Rotated Matrix:")
for i in range(0,n):
    for j in range(0,n):
        print(ar[i][j]," ",end="")
    print("\n")
